conneg-by-ap/index.html

```diff
@@ -841,6 +841,43 @@ <h3>Alternate Representations Data Model</h3>
       </p>
     </section>
   </section>
+  <section id="link-attributes">
+    <h2>Link Attributes</h2>
+      <section id="link-attributes-token">
+	<h3>Token</h3>
+	<p>
+	  When used in a <code>Link</code> header field,
+	  the attribute <code>token</code> is used to specify
+	  the mapping between the profile URI
+	  (specified using the <code>anchor</code> attribute)
+	  and a <a>token</a> used as an alternative to the full profile URI.
```

Comment:
Minor comment, but perhaps the "mapping" language does not need to end up in the document. We talk about this mapping in the context of this WG, but the concept itself is probably not important to the document. So it could be fine to just write that the token is a (typically shorter) string that can be used by clients as an alternative to the full profile URI.

Reply:
Are you proposing that the document just says "token is a (typically shorter) string that can be used by clients as an alternative to the full profile URI" and then examples, like the one you submitted in your earlier PR, carry the demonstration? I'd prefer this written explanation be included but perhaps I've missed what you're suggesting.

Reply:
Yes, that's what I am proposing. Along with the removal of the "mapping" language.

Reply:
Yes, that's what I am proposing. Along with the removal of the "mapping" language.